exercises/practice/matrix/.docs/instructions.append.md

```diff
@@ -0,0 +1,20 @@
+# Instructions append
+
+## Challenges
+
+- Can you implement this solution with using **ANY** `for` loops? (ie using [iterators][iterators] instead)
```

Comment:
```suggestion
- Can you implement this solution without using any `for` loops? (i.e. using [iterators][iterators] instead)
```

Reply:
I'm a dummy, I forgot to add `no important changes`. I'll remember next time, sorry.

Reply:
You don't need to worry about that. It's only needed when updating an existing exercise, so it's irrelevant for this PR. There is also an automation workflow which comments on PRs to remind people when it's relevant. Lastly, we always squash PRs and the `[no important files changed]` label only matters on those squash commits - so, when merging a PR.